selinux/ipa.te

```diff
@@ -121,6 +121,9 @@ optional_policy(`
 	sssd_stream_connect(ipa_otpd_t)
 ')
 
+dev_read_sysfs(ipa_otpd_t)
```

Comment:
Can you move the new lines up close to the previous dev_*?

Reply:
done